protocol/chainlib/tendermintRPC.go

```diff
@@ -863,7 +806,10 @@ func (cp *tendermintRpcChainProxy) SendRPC(ctx context.Context, nodeMessage *rpc
 		}
 		subscriptionID, ok = paramsMap["query"].(string)
 		if !ok {
-			return nil, "", nil, utils.LavaFormatError("unknown subscriptionID type on tendermint subscribe", nil)
+			utils.LavaFormatTrace("could not get subscriptionID from query params", utils.LogAttr("params", params))
+			// This is probably because of a misuse, return error
+			subscriptionID = ""
+			// return nil, "", nil, utils.LavaFormatError("unknown subscriptionID type on tendermint subscribe", nil)
```

Comment:
error is commented out, is this a mistake?

Reply:
This PR was WIP, so you might see some unexplained comments here and there. I think you can get rid of this one.